pallets/runtime/common/src/bridge.rs

```diff
@@ -526,9 +524,10 @@ decl_module! {
             Self::handle_signed_bridge_tx(&sender, bridge_tx)
         }
 
-        /// Handles a vector of approved bridge transaction proposals. The vector is processed until
-        /// the first proposal which causes an error, in which case the error is returned and the
-        /// rest of proposals are not processed.
+        /// Handles a vector of approved bridge transaction proposals.
+        /// Forces handling a vector of transactions by bypassing the bridge limit and timelock.
+        /// It collects results of processing every transaction in the given vector and outputs
+        /// the vector of results (In event) which has the same length as the `bridge_txs` have
```

Comment:
Perhaps it would be clearer to say "deposits an event with the vector of reaults" instead of "outputs" and "(In event)".